modules/localgov_search_db/tests/Functional/SitewideSearchTest.php

```diff
@@ -19,6 +19,7 @@ class SitewideSearchTest extends SitewideSearchBase {
   protected static $modules = [
     'localgov_search',
     'localgov_search_db',
+    'big_pipe',
```

Comment:
Is this needed? It's a but strange if it is. Just to check, I deleted this line and the tests ran fine.

Reply:
It was part of my trying to find something getting cached incorrectly for https://github.com/localgovdrupal/localgov_search/pull/47 I figured big_pipe could cause additional caching issues. Still not managed to reproduce the problem but left it in there to add things that might break the tests.